src/ssr-template-helper.js

```diff
@@ -72,7 +72,7 @@ export class SsrTemplateHelper {
    *     the payload. If provided, finding the template in the passed in
    *     element is not attempted.
    * @param {!Object=} opt_attributes Additional JSON to send to viewer.
-   * return {!Promise<{data:{?JsonObject|string|undefined}}>}
```

Comment:
This doesn't look correct.

Reply:
https://github.com/ampproject/amphtml/pull/23559/commits/9ba45dfc5e971f8531efb8cd7361712d6d0e5077#diff-e0a7330b8f95754a8f79dd9d6eeb035c